modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -7,10 +7,10 @@ This document helps you use Object Store v2 (OSv2) to store and view data.
 
 == Prerequisites
 
-This document assumes you are familiar with Anypoint Studio, using curl from a command line, 
+This document assumes you are familiar with Anypoint Studio, using https://curl.haxx.se/[curl] from a command line, 
 and the use of Anypoint Platform and Runtime Manager. Ensure that curl is installed on 
 your computer. If you are using a Mac or Linux, the curl command is built in. For Windows, 
-download and install curl from the net.
+download and install curl from the net. Alternatively, you can use Postman or browser plugins instead of curl.
```

Comment:
can we just say download and install curl? and remove **from the net**

Reply:
Fixed as requested.